tests/evaluator/test_text_classification.py

```diff
@@ -0,0 +1,73 @@
+import json
+import os
+import subprocess
+import unittest
+from datetime import datetime
+
+from datasets import load_dataset
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, pipeline
+
+from evaluate import evaluator
+
+
+class TestTextClassificationEvaluator(unittest.TestCase):
+    def test_match_transformers_example(self):
+        model_name = "howey/bert-base-uncased-sst2"
+
+        tps = datetime.now().isoformat()
+        dir_path = f"/tmp/{tps}_textclassification"
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+        os.chdir(dir_path)
```

Comment:
A slightly nicer way is to use to pass `cwd=dir_path` to `suprocess.run`.

Reply:
Indeed, I did the change!